### DISCUSSION TITLE
New tab

### DISCUSSION THREAD

**Discussion by Gottarocket** [Docile-Alligator/Infinity-For-Reddit#1037] [Ideas]
Option to show new tab is needed as we would be checking between subreddits and want to have previous comment or post to be accessible when searching another so if there is a tab option we can search within 2 or more subreddits at once 
 Freetube has tabs to open different videos on different tabs but its basedon chromium so not sure if it can be pulled here but would appreciate it

**Comment by yephny** [comment]
Windows like [Material Files](https://github.com/zhanghai/MaterialFiles) has would work as well.

**Comment by Gottarocket** [comment]
Yes but would appreciate tab if possible on app instead of accessing recent and going to new window

**Comment by tbertels** [comment]
I'm guessing you mean the ability to open posts in new tabs?
It would allow Infinity for Reddit to fully replace the browser for Reddit and avoid its obnoxious "better viewed in the app" banners.
